src/components/App/index.tsx

```diff
@@ -37,6 +37,31 @@ export function App() {
     })
   }
 
+  const editTask = (id: string, text: string) => {
```

Comment:
It looks like when a task is edited, it automatically goes to the bottom of the list. Can you think of a way we could keep them in the same order?

Reply:
Resolved in the most recent push!